clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -357,12 +357,8 @@ check_kernel_xfer()
     // Return to recorded interruption point.
```

Comment:
The PR description is missing a `Fixes #6288` or `Issue: #6288` line (which one depends on whether it should close the issue).